### PR TITLE
Nginx docker-compose, PVWatts bug fix

### PR DESCRIPTION
**Pull request by NLaws**
### Patches
- `reo`: Fix **PVWatts** being called when user provides `PV.prod_factor_series_kw`
- `reopt_api`: new `docker-compose.nginx.yml` for standing up the API on a server with remote access (for example if one wants to host the API on a cloud service)
- `reopt_api`: update `Dockerfile.xpress` to use `nlaws/pyjul:1.5.2` base image (was using Julia 1.3)

### CODE REVIEW

File: reo/src/techs.py [hunk 1]
```diff
@@ -146,7 +148,10 @@ def __init__(self, dfm, degradation_pct, time_steps_per_hour=1, acres_per_kw=6e-
             else:  # All other tilts come from lookup table included in the array_type_to_tilt_angle dictionary above
                 self.tilt = PV.array_type_to_tilt_angle[kwargs.get('array_type')]
 
-        self.pvwatts = PVWatts(time_steps_per_hour=self.time_steps_per_hour, azimuth=self.azimuth, tilt=self.tilt, **self.kwargs)
+        if self.prod_factor_series_kw is not None:  # then don't call PVWatts
+            self.station = (kwargs.get("latitude", 0), kwargs.get("longitude", 0), 0)
```
Comment (t-kwasnik) [inline reo/src/techs.py:152]: Do we need to set station coords? Lat and long are required so we shouldn't get (0,0), but that is is a real point and may be confusing if it comes back from the API. Is there a problem with (None, None)?

File: reo/src/pvwatts.py [hunk 2]
```diff
@@ -48,7 +48,8 @@ def check_pvwatts_response_data(resp):
         message = ("Error parsing PVWatts response. "
                    "Status code: {}. Text: {}".format(resp.status_code, resp.text))
         raise_pvwatts_exception(message)
-    return data
+    else:
```
Comment (t-kwasnik) [inline reo/src/pvwatts.py:51]: Just curious, why was this change needed? If an exception is encountered we'll exit out of this workflow through the raise funciton

**Review by t-kwasnik — COMMENTED** [review]
@NLaws left two quick questions, can approve soon after you get to those

File: reo/src/pvwatts.py (hunk 2 above)
Comment (NLaws) [reply reo/src/pvwatts.py:51]: Two reasons: 
1. clarity: the `else` statement says if no exception occurs then do this
2. robustness: if the `except` block is ever changed to something that does not raise an exception then the code would fail (since `data` would not be defined due to the scoping of the try/except block)

File: reo/src/techs.py (hunk 1 above)
Comment (NLaws) [reply reo/src/techs.py:152]: I was trying to preserve old behavior, but we don't need to set them. (Old behavior was to call PVWatts to get station lat/lon even when user provider their own prod_factor).

**Comment by NLaws** [comment]
the Julia upgrades might fix https://github.com/NREL/REopt_Lite_API/issues/183 (I have run 3,000 jobs on this branch and seen no lost jobs), which include:
- Julia 1.5.3 available in nlaws/pyjul images (we will have to manually upgrade on RHEL servers)
- PyCall.jl 1.91.4 >> 1.92.2

File: reo/src/techs.py (hunk 1 above)
Comment (t-kwasnik) [reply reo/src/techs.py:152]: @NLaws thanks - I'll approve this and check with Andrew about the 0,0 issue and decide how to handle that after that

**Review by t-kwasnik — APPROVED** [review]
